content/web/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Web WorldWind"
+date: 2017-07-16T21:40:51-05:00
+draft: false
+mainpage: false
+projectpage: true
+projectslug: "web"
+projectname: "Web WorldWind"
+listdescription: ""
+---
+
+## Get Started
+
+Web WorldWind is a 3D virtual globe API for HTML5 and JavaScript. Web pages include Web WorldWind to provide one or more virtual globes on that page. The [European Space Agency](https://www.esa.int/) is now partnering with NASA on development. Work on Web WorldWind began in 2014 and is very active today.
+
+---
+
+## Download
+
+*The first release of Web WorldWind is imminent, please check back soon for updates to these instructions! The versioned release will be the first official release of Web WorldWind and will be available as a hosted library and through NPM.*
+
+To add Web WorldWind to your web application, build the worldwind.js and worldwind.min.js files locally and host the the library in a manner which best supports your applications requirements. To build the worldwind.js and worldwind.min.js files in your own environment follow the directions in the [HowToBuildWebWW](https://github.com/NASAWorldWind/WebWorldWind/blob/develop/HowToBuildWebWW.md) document in the repository.
```

Comment:
Could this be expressed as a simple list? It's wordy and difficult to make out what actions are needed. The result sounds a lot more complex than it is.